packages/table/test/formatsTests.tsx

```diff
@@ -69,7 +71,7 @@ describe("Formats", () => {
                 majority have never stirred?
             `;
 
-            const style = { height: "200px" };
+            const style = { height: "200px", position: "relative" };
```

Comment:
i dont entirely understand why this would have broken or why this change fixes it but it's possible some deeply nested SCSS rule became invalid with the new DOM hierarchy.

if so, we should add this css rule to the TABLE_TRUNCATED_TEXT class instead of just the style on the test